includes/wikia/services/tests/CommunityDataServiceTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+class CommunityDataServiceTest extends WikiaBaseTest {
+	const FEATURED_SECTION = 'featured';
+	const CURATED_SECTION = 'curated';
+	const OPTIONAL_SECTION = 'optional';
+	const COMMUNITY_DATA = 'community_data';
```

Comment:
Why not reuse consts from CommunityDataService, i.e. CommunityDataService:: FEATURED_SECTION ?